contributing_to_docs/doc_guidelines.adoc

```diff
@@ -20,6 +20,10 @@ It is not necessary to update existing content to unwrap lines, but you can remo
 In the Atom editor, you can use `Ctrl`+`J` to undo hard wrapping on a paragraph.
 ====
 
+* The following books are auto-generated and should not be manually edited:
```

Comment:
I'm not sure if this is the right place for this guidance, since this is general guidelines on how to edit files.

I'd consider adding it as it's own new section on auto-generated content. wdyt?

Reply:
Yes agreed good idea